projects/plugins/jetpack/tests/php/general/test_jetpack-admin-menu.php

```diff
@@ -95,8 +95,10 @@ public function test_jetpack_admin_menu_order() {
 			$this->assertTrue( $activity_log_submenu_position < $search_submenu_position, 'Activity Log should be above Search in the submenu order.' );
 			$this->assertTrue( $backup_submenu_position < $activity_log_submenu_position, 'Jetpack VaultPress Backup should be above Activity Log in the submenu order.' );
 		}
-
-		$this->assertTrue( $stats_submenu_position < $videopress_submenu_position, 'Stats should be above VideoPress in the submenu order.' );
+		// The VideoPress menu item may load late and affect this particular test.
```

Comment:
This seems odd. Why does only this one fail?

Also, perhaps we should switch all these tests to [`assertLessThan()`](https://docs.phpunit.de/en/10.5/assertions.html#assertlessthan) since we're here?

Reply:
Ah, probably this should be changed to call the new `->enable_menu()` instead.
https://github.com/Automattic/jetpack/blob/a662c4d95c5e3464e1cefe8104068c20e7ff0f95/projects/plugins/jetpack/tests/php/general/test_jetpack-admin-menu.php#L63